### DISCUSSION TITLE
implementation of global pt not working since some primevue updates

### DISCUSSION THREAD

**Discussion by domutom** [primefaces/community#98] [PrimeVue]
I've implemented primevue exactly like you suggesting in your docs and it worked until some latest updates of primevue. I did all kinds of implementation with/without tailwind - could only define pt usage in each single UI component - this works. At the time it works in my case I could define global pt component s removing tailwind classes which I have to set in single components. Overwriting never worked, but this solution was working brilliantly for me

**Comment by tugcekucukoglu** [comment]
I want to share global pt implementation example.

Configuration with Tailwind:

```
import {createApp} from "vue";
import PrimeVue from "primevue/config";
import { usePassThrough } from "primevue/passthrough";
import Tailwind from "primevue/passthrough/tailwind";

const app = createApp(App);

const CustomTailwind = usePassThrough(
    Tailwind,
    {
        panel: {
            ****
        }
    },
);

app.use(PrimeVue, { unstyled: true, pt: CustomTailwind });
```



Configurarion without Tailwind:

```
import {createApp} from "vue";
import PrimeVue from "primevue/config";

const app = createApp(App);

const CustomTheme = {
        panel: {
            ***
        }
  };

app.use(PrimeVue, { unstyled: true, pt: CustomTheme });
```

Which version of PrimeVue are you using? And could share how did you try to use global pt?

**Comment by domutom** [comment]
Hi, thanks for the reply. Will check with newest version you released today. implementation like you did is not working in my case. Need to investigate a bit more
